js/lib/ilib.js

```diff
@@ -129,7 +129,7 @@ ilib._getPlatform = function () {
     		}
     	} catch (e) {}
     	
-        if (typeof(process) !== 'undefined' && typeof(window) === 'undefined' && typeof(module) !== 'undefined') {
```

Comment:
Yes, the tests with the code as it was for Enact did not work with the regular nodejs tests without Enact. The reason is that the tests are run inside of a nodejs "vm" which has a "window" variable in its context. (See https://nodejs.org/api/vm.html for reference documentation.) Many tests failed because of this, even though the code itself was fine. Ilib needs to find some other way of detecting Enact as a platform.